dags/common/ingestion.py

```
@@ -0,0 +1,149 @@
+        lat (float): 위도
+        lon (float): 경도
+    Returns:
+        Dict[str, Any]: API 응답 결과 JSON 데이터
+    Raises:
+        ValueError: API 키가 환경 변수에 존재하지 않을 경우
+        Exception: API 응답 상태 코드가 200이 아닐 경우
+    """
+    api_key = os.getenv("OPENWEATHER_API_KEY")
+    if not api_key:
+        logger.error("Environment variable 'OPENWEATHER_API_KEY' is missing")
+        raise ValueError("OPENWEATHER_API_KEY is not exist")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)